commit 0affa5c5ed8d0f225525fc9da6d6dc0b55b0e6fd (refs/changes/38/2738/3)
Author: Todd Whiteman <todd.whiteman@joyent.com>
Date:   2017-10-13T12:59:00-07:00 (2 years ago)
    
    DOCKER-592 disable the restify uncaughtException handler
    Reviewed by: Julien Gilli <julien.gilli@joyent.com>
    Approved by: Julien Gilli <julien.gilli@joyent.com>

diff --git a/lib/docker.js b/lib/docker.js
index 2ee0fd0..31c0e6a 100644
--- a/lib/docker.js
+++ b/lib/docker.js
@@ -279,6 +279,7 @@ App.prototype.setupServer = function () {
         log: self.log,
         name: 'sdc-docker',
         version: self.version,
+        handleUncaughtExceptions: false,
         handleUpgrades: true,
         formatters: {
             /*
diff --git a/package.json b/package.json
index 9bef42e..5da0549 100644
--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "sdc-docker",
-  "version": "0.4.8",
+  "version": "0.4.9",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
